paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/EventReporter.kt

```diff
@@ -7,49 +7,99 @@ import com.stripe.android.paymentsheet.model.PaymentSelection
 
 internal interface EventReporter {
 
+    /**
+     * PaymentSheet has been instantiated or FlowController has finished its configuration.
+     */
     fun onInit(
         configuration: PaymentSheet.Configuration?,
         isDecoupling: Boolean,
     )
 
+    /**
+     * PaymentSheet or FlowController have started loading.
+     */
+    fun onLoadStarted(
+        isDecoupling: Boolean,
+    )
+
+    /**
+     * PaymentSheet or FlowController have successfully loaded the information required to be
+     * rendered.
+     */
+    fun onLoadSucceeded(
+        isDecoupling: Boolean,
+    )
+
+    /**
+     * PaymentSheet or FlowController have failed to load.
+     */
+    fun onLoadFailed(
+        isDecoupling: Boolean,
```

Comment:
This should eventually also include the error type, but we need to do some other work first to create these strings that are safe to log.